test/TestExpectations.json

```diff
@@ -537,7 +537,7 @@
     "testIdPattern": "[coverage.spec] Coverage specs JSCoverage should ignore pptr internal scripts if reportAnonymousScripts is true",
     "platforms": ["darwin", "linux", "win32"],
     "parameters": ["webDriverBiDi"],
-    "expectations": ["FAIL", "PASS"]
+    "expectations": ["FAIL"]
```

Comment:
This is incorrect due to https://github.com/GoogleChromeLabs/chromium-bidi/issues/896 making it flaky